code/parachain/frame/fnft/src/test/mock.rs

```diff
@@ -8,14 +8,21 @@ use frame_support::{
 	PalletId,
 };
 use frame_system as system;
-use sp_core::H256;
+use frame_system::EnsureRoot;
+pub use sp_core::{
+	sr25519::{Public, Signature},
+	H256,
+};
 use sp_runtime::{
 	testing::Header,
 	traits::{BlakeTwo256, IdentityLookup},
 };
 
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<MockRuntime>;
+type AccountId = u128;
+type Balance = u128;
 type Block = frame_system::mocking::MockBlock<MockRuntime>;
+type CurrencyId = u128;
```

Comment:
I would just use `u128` directly in the configs, type aliases can be confusing at use sites because they can get confused with newtypes (while not providing the safety of newtypes)

Reply:
I understand where your coming from, but this also makes updating the types easier. There are certainly tradeoffs either way.